src/nvme/types.h

```diff
@@ -5997,6 +5999,75 @@ enum nvme_apst_entry {
 	NVME_APST_ENTRY_ITPT_MASK = 0xffffff,
 };
 
+/**
+ * struct nvme_std_perf_attr - Standard performance attribute structure
+ * @rsvd0:	Reserved
+ * @r4karl:	Random 4 KiB average read latency
+ * @rsvd5:	Reserved
+ */
+struct nvme_std_perf_attr {
+	__u8 rsvd0[4];
+	__u8 r4karl;
+	__u8 rsvd5[4091];
+};
+
+/**
+ * struct nvme_perf_attr_id - Performance attribute identifier structure
+ * @id:	Performance attribute identifier
+ */
+struct nvme_perf_attr_id {
+	__u8 id[NVME_UUID_LEN];
+};
+
+/**
+ * struct nvme_perf_attr_id_list - Performance attribute identifier list structure
+ * @attrtyp:	Bits 7-3: Reserved
+ *		Bits 2-0: Attribute type
+ * @msvspa:	Maximum saveable vendor specific performance attributes
+ * @usvspa:	Unused saveable vendor specific performance attributes
+ * @rsvd3:	Reserved
+ * @id_list:	Performance attribute identifier list
+ * @rsvd1024:	Reserved
+ */
+struct nvme_perf_attr_id_list {
+	__u8 attrtyp;
+	__u8 msvspa;
+	__u8 usvspa;
+	__u8 rsvd3[13];
+	struct nvme_perf_attr_id id_list[1008];
```

Comment:
Isn't the range from [0xc, 0xff], so 63?

Reply:
Yes sorry for the error.